freqtrade/freqtradebot.py

```diff
@@ -17,6 +17,7 @@
 from freqtrade import (DependencyException, OperationalException,
                        TemporaryError, __version__, constants, persistence)
 from freqtrade.exchange import Exchange
+from freqtrade.finance.wallets import Wallets
```

Comment:
i would move the wallets file to the base folder (`freqtrade/wallets.py`) instead of adding another submodule with 1 file in it.  
we can always refactor this into a package later if this becomes a bigger module with multiple classes ...